app-deploy.sh

```diff
@@ -108,6 +109,23 @@ function initial_checkup {
     fi
 }
 
+function input_to_tags {
+
+    # Parse all selected options
+    IFS=', ' read -r -a environments_array <<< "$target_selection"
+
+    environments_to_build=()
+
+    for environment in "${environments_array[@]}"; do
+        if [ ${environment} -le 9 -a ${environment} -ge 0 ]; then
+            environments_to_build+=("${environments[${environment}]}")
+        else
```

Comment:
Why do we check 0-9 range, I mean, what if there is 10+ envs? Not common for most of projects, but could happen. Should we just check if the environment number is not larger of the possible env count? Sry if I misunderstood the logic.

Reply:
Oh, you are right! 

I totally missed that one. I'll try to extract a number of items in the array and then check between 0 and the array.count.
If Bash doesn't have this, I'll remove this check entirely.

Reply:
Yeah, I think we can use`${#arrayName[@]}` for that :)

Reply:
I had to add `$((${#environments[@]} - 1))` on that place to get max - 1 (counting from 0).